### DISCUSSION TITLE
Lagrange.Core 如何发起临时会话，向群成员发送私聊消息？

### DISCUSSION THREAD

**Discussion by sectlying** [LagrangeDev/Lagrange.Core#421] [Q&A] [answered]
![41aed72841cb1166027c74faea7312ea](https://github.com/LagrangeDev/Lagrange.Core/assets/77864650/789699ea-b286-4e5e-b37a-c3541fd78dbe)

如图为相似的需求在Mirai的写法（收到群聊消息事件后，发一条群聊消息，再发一条临时会话消息）

**Comment by ghost** [comment, marked as answer]
Lagrange.Core 不支持也不会支持发送临时会话

**Reply by sectlying** [reply]
能说说原因吗？
